posts/posts.robot

```diff
@@ -0,0 +1,75 @@
+*** Setting ***
+Resource    ./post_resources.robot
+Test Setup    Create HTTP Context     ${HOST}
```

Comment:
Create a keyword for the setup where you include the context and the request headers used on the cases below so you avoid some repetition. Something like:
Setup keyword
      Context
      Headers...